File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

async function readPdfs() {
    console.log('Reading first PDF...');
    let dataBuffer1 = fs.readFileSync('c:\\Users\\aizpr\\vetechi-landing\\VETECHI_Documentacion_Institucional_2026.pdf');
    let data1 = await pdf(dataBuffer1);
    console.log('--- VETECHI_Documentacion_Institucional_2026.pdf ---');
    console.log(data1.text);
    console.log('\n\n======================================================\n\n');

    console.log('Reading second PDF...');
    let dataBuffer2 = fs.readFileSync('c:\\Users\\aizpr\\vetechi-landing\\VETECHI · Hospital Veterinario Chiriquí · Emergencias 24_7.pdf');
    let data2 = await pdf(dataBuffer2);
    console.log('--- VETECHI · Hospital Veterinario Chiriquí · Emergencias 24_7.pdf ---');
    console.log(data2.text);
}

readPdfs().catch(console.error);

File: read_pdf.py
import sys
from pypdf import PdfReader

sys.stdout.reconfigure(encoding='utf-8')

print("Reading first PDF...")
try:
    reader1 = PdfReader('VETECHI_Documentacion_Institucional_2026.pdf')
    text1 = ""
    for page in reader1.pages:
        text1 += page.extract_text() + "\n"
    print("--- VETECHI_Documentacion_Institucional_2026.pdf ---")
    print(text1)
except Exception as e:
    print("Error reading first PDF:", e)

print("\n\n======================================================\n\n")

print("Reading second PDF...")
try:
    reader2 = PdfReader('VETECHI · Hospital Veterinario Chiriquí · Emergencias 24_7.pdf')
    text2 = ""
    for page in reader2.pages:
        text2 += page.extract_text() + "\n"
    print("--- VETECHI · Hospital Veterinario Chiriquí · Emergencias 24_7.pdf ---")
    print(text2)
except Exception as e:
    print("Error reading second PDF:", e)
